src/components/WebexRemoteMedia/WebexRemoteMedia.scss

```diff
@@ -26,5 +26,17 @@
   video {
     height: 100%;
     width: 100%;
+
+    &.share {
```

Comment:
Is something using this?

Reply:
Yes, it's part of remoteShareClasses used to control the visibility of sharing video (not the cam video).

Reply:
This part is also changed for the removal of `remoteShareClasses`.